testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java

```diff
@@ -1,6 +1,5 @@
 package org.jboss.as.test.integration.management.cli;
 
-import com.sun.javafx.util.Utils;
```

Comment:
do you know if that the only use of JavaFX in the test suite?
Could we also clean up the JavaFX dependency in the pom?

Reply:
That is an invalid import, really caused by "automatic import resolution of IDE" and overlook at it. Removing from pom, I can't say. Didn't find other occurrences in tests.